core/arch/arm/kernel/unwind_arm64.c

```diff
@@ -96,3 +97,43 @@ void print_kernel_stack(int level)
 }
 
 #endif
+
+#if defined(CFG_UNWIND)
```

Comment:
Useless `#ifdef CFG_UNWIND`. The same comment applies to change in **unwind_a32.c**. 
Feel free to address or not this comment...

Reply:
OK, and I will also clean up the others in a separate commit.